packages/client/hmi-client/src/components/workflow/ops/optimize-ciemss/tera-static-intervention-policy-group.vue

```diff
@@ -51,27 +69,58 @@
 				<section v-if="showNewValueOptions">
 					<h6 class="pt-4, pb-3">New Value</h6>
 					<div class="input-row">
-						<div v-for="(objective, index) in newValueInputs" :key="index" class="label-and-input">
-							<div class="label-and-input">
-								<label :for="objective">{{ NEW_VALUE_OPTIONS[index] }}</label>
-								<tera-input type="number" v-model="knobs[objective]" />
-							</div>
+						<div class="label-and-input">
+							<label>Lower bound</label>
+							<tera-input
+								type="number"
+								v-model="knobs.lowerBoundValue"
+								@update:model-value="$emit('update-self', knobs)"
+							/>
+						</div>
+						<div class="label-and-input">
+							<label>Upper bound</label>
+							<tera-input
+								type="number"
+								v-model="knobs.upperBoundValue"
+								@update:model-value="emit('update-self', knobs)"
+							/>
+						</div>
+						<div class="label-and-input">
+							<label>Initial guess</label>
+							<tera-input
+								type="number"
+								v-model="knobs.initialGuessValue"
+								@update:model-value="emit('update-self', knobs)"
+							/>
 						</div>
 					</div>
 				</section>
-
 				<section v-if="showStartTimeOptions">
 					<h6 class="pt-4, pb-3">Start Time</h6>
 					<div class="input-row">
-						<div
-							v-for="(objective, index) in newStartTimeInputs"
-							:key="index"
-							class="label-and-input"
-						>
-							<div class="label-and-input">
-								<label :for="objective">{{ NEW_VALUE_OPTIONS[index] }}</label>
-								<tera-input type="number" v-model="knobs[objective]" />
-							</div>
+						<div class="label-and-input">
+							<label>Start time</label>
+							<tera-input
+								type="number"
+								v-model="knobs.startTime"
+								@update:model-value="emit('update-self', knobs)"
+							/>
+						</div>
```

Comment:
@blanchco added a `label` props to `tera-input`.
Also do we need those `<div class="lable-and-input">`?

```suggestion
						<div class="label-and-input">
							<tera-input
								type="number"
								:label="Start time"
								v-model="knobs.startTime"
								@update:model-value="emit('update-self', knobs)"
							/>
						</div>
```